autowarefoundation/autoware.universe
# fix(behavior_velocity_planner.stopline): extend following and previous search range to avoid no collision

## Description

- extend +1 following and previous search range to avoid no collision
- ~~add debug marker to confirm this module ( I'll remove when the review is over)~~



## Related links



## Tests performed
Before
![image](https://user-images.githubusercontent.com/9547070/169189624-5bd115c3-e0fa-437f-8439-2ef6f0b83d9c.png)

After
![image](https://user-images.githubusercontent.com/9547070/169192756-928423a0-db01-4bb0-8e71-b4b09b62159d.png)


In planning_simulator...

1. add start and goal near stopline like above image
2. add a dummy obstacle near stopline like above image
3. engage
4. When the path is near the dummy obstacle, behavior_path_planner change module, `lane_following` to `Avoidance`
5. Before: Stopline is disappeared like before image
6. After: Stopline keeps to appear like after image

## Notes for reviewers



## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.


## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.


## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.


After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/
